### PR TITLE
Add support for Swig.

### LINKED ISSUE #711: [Icon Request] Support for SWIG
- [x] I'm sure this issue is not a *duplicate*.
- [x] I want to create an icon request:
    - Type: `extension`
    - Icon Name: `swig`
    - Sample original Icon: N/A
    - Extensions: `.swig`

- More info @ http://node-swig.github.io/swig-templates/

### PR DESCRIPTION
**Pull request by JimiC**
_**Fixes #711**_

**Changes proposed:**

- [x] Add

### CODE REVIEW

**Comment by JimiC** [comment]
The colors where chosen to represent:
- `NodeJS` (green)
- `JS` (yellow-orange)

File: src/icon-manifest/supportedExtensions.ts [hunk 1]
```diff
@@ -488,6 +488,7 @@ export const extensions: IFileCollection = {
     { icon: 'svg', extensions: ['svg'], format: FileFormat.svg },
     { icon: 'swagger', extensions: [], languages: [languages.swagger], format: FileFormat.svg },
     { icon: 'swift', extensions: ['package.pins'], filename: true, languages: [languages.swift], format: FileFormat.svg },
+    { icon: 'swig', extensions: [], languages: [languages.swig], format: FileFormat.svg },
```
Comment (jens1o) [inline src/icon-manifest/supportedExtensions.ts]: where's the definition of that language?

Comment (JimiC) [reply src/icon-manifest/supportedExtensions.ts]: Already in.

Comment (jens1o) [reply src/icon-manifest/supportedExtensions.ts]: How could we missed that?

Comment (JimiC) [reply src/icon-manifest/supportedExtensions.ts]: Missed?

Comment (jens1o) [reply src/icon-manifest/supportedExtensions.ts]: I mean like: Why do we defined the language without the icon and the entry in our collection?

Comment (JimiC) [reply src/icon-manifest/supportedExtensions.ts]: Cause there was no icon at that time. I just found the time now to create an icon for it.

Comment (jens1o) [reply src/icon-manifest/supportedExtensions.ts]: Ah, sounds reasonable. Thank you!

**Pull request merged**